### ISSUE TITLE
idle_inhibitor stopped working

### ISSUE THREAD

**Issue by RononDex** [Alexays/Waybar#3268]
Since recently the idle_inhibitor stopped preventing swayidle and hypridle (tested both) from triggering.

Tested on hyprland & Arch Linux

**Comment by TheoSmith0308** [comment]
Same issue. The idle_inhibitor fails to stop hypridle. This is on arch hyprland with waybar.

**Comment by Zerron21** [comment]
I can report that it also doesn't work with swayidle on hyprland either, not sure when it stopped working exactly but it had to have been about a week ago.

**Comment by mostafaqanbaryan** [comment]
> I can report that it also doesn't work with swayidle on hyprland either, not sure when it stopped working exactly but it had to have been about a week ago.

Can confirm. With hyprland and swayidle.

So the problem is not swayidle or hypridle.

**Comment by Lynthium** [comment]
Ran into this as well, some digging later, it's likely this issue: https://github.com/hyprwm/Hyprland/issues/5878
Fix has been merged so next Hyprland v0.41.0 release should have it (no idea when that's scheduled to release though).

**Comment by TheoSmith0308** [comment]
Yes it is a Hyprland issue. If you compile Hyprland from Github it works but I experienced other funny effects with the compiled version so I reverted to the regular Arch one. I hope they fix it soon. Currently I just dont run hypridle at startup.

**Comment by Alexays** [comment]
Closing as outdated — this was an upstream Hyprland bug (hyprwm/Hyprland#5878), fixed and shipped in Hyprland v0.41.0. Please update Hyprland; reopen if it still occurs. 🙏
